content/security/for-maintainers.adoc

```diff
@@ -123,6 +123,40 @@ This strikes a balance between allowing the longest possible time for reviews an
 Make sure to follow instructions provided by the Jenkins security team if they deviate from the instructions below.
 
+[[backporting]]
+=== Backporting
+
+At this point, consider the need for backports.
+Some administrators may be unable to install the latest version of your plugin.
+Usually this is because the plugin depends on a recent weekly release of Jenkins.
+Users of the link:/download/lts/[LTS line of Jenkins releases] will not be offered a plugin update that requires a weekly version of Jenkins higher than the LTS baseline, even when they're already on the latest version of Jenkins available to them.
+
+We recommend that all security fixes are made available to both current weekly and current LTS releases.
+If the current LTS release is the first in its line (e.g., 2.375.1), also consider making the security fix available to users of the previous release, as many administrators may not immediately update.
+
+Additionally, the security fixes should generally be made available to users who are already on the latest version of the plugin available to them, rather than an earlier version.
+This will make it easy to update to a version with the fix through the plugin manager, without potentially losing features by doing what amounts to a manual downgrade.
+
+
+==== Backporting Example
+
+Suppose it is early 2023 and you're preparing a security fix for a plugin you maintain.
+The most recent LTS releases are Jenkins 2.361.4 and Jenkins 2.375.1, and the current weekly release is 2.387.
+The plugin's latest release, version 2.2, has a Jenkins dependency of 2.380 because it makes use of a new feature in that release.
+The most recent lower Jenkins dependency of the plugin was Jenkins 2.361.3 in plugin version 2.0.
+
+In this case, instances on both 2.361.4 and 2.375.1 will be able to install a security fix provided on top of version 2.0 of the plugin.
+Instances on the weekly release line will be able to install a security fix provided on top of version 2.2.
+So the plugin should get updates with the security fix based on top of version 2.2 (e.g., 2.2.1 or 2.3) _and_ 2.0 (e.g., 2.0.1 or 2.0.0.1, whichever version would be between 2.0 and the next release that already exists).
+
+Now consider the case of the plugin version 2.1 previously raising the core dependency from 2.361.2 to Jenkins 2.370 (before 2.2 raised it to 2.380).
+In this care, three separate releases are now needed to cover users of Jenkins 2.361.4, 2.375.1, and 2.387:
+
+* Jenkins 2.361.4 needs a backport on top of plugin version 2.0 (e.g., 2.0.1 or 2.0.0.1) with a Jenkins dependency of 2.361.2.
+* Jenkins 2.375.1 needs a backport on top of plugin version 2.1 (e.g., 2.1.1 or 2.1.0.1) with a Jenkins dependency of 2.370. While they could in theory install the backport on top of version 2.0, this would effectively be a downgrade, and remove all features added between versions 2.0 and 2.1 of the plugin (if this is even possibly without data loss).
```

Comment:
```suggestion
* Jenkins 2.375.1 needs a backport on top of plugin version 2.1 (e.g., 2.1.1 or 2.1.0.1) with a Jenkins dependency of 2.370. While they could in theory install the backport on top of version 2.0, this would effectively be a downgrade, and remove all features added between versions 2.0 and 2.1 of the plugin (if this is even possible without data loss).
```